push_latest.sh

```diff
@@ -0,0 +1,103 @@
+source venv/bin/activate
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+# -----------------------------------------------------------------------------
+# Validate and build the release.
+# -----------------------------------------------------------------------------
+
+# Affirm on the master branch
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+if [ "$BRANCH" != "master" ]; then
+    echo -e "${RED}Not on master branch, run:${NC}"
+    echo "git checkout master"
+    exit
+fi
+
+# TODO(TheDuckCow): Perform check when we have an automated way to set prod env.
+# if [[ `git status --porcelain` ]]; then
+#   echo "There are uncommited changes, ending"
+#   exit
+# fi
+
+git pull --quiet
+echo ""
+echo "Current status (should be empty!)"
+git status
+
+echo "Running tests"
+python run_tests.py -a
+if [ $? -eq 0 ]; then
```

Comment:
Assuming this is Bash, you can check the command exit code directly 
```suggestion
if "python run_tests.py -a"; then
```

Reply:
Unforrunately when I tried this on my machine, the script errors with:

`./push_latest.sh: line 32: python run_tests.py -a: command not found`